ports/servoshell/egl/servo_glue.rs

```diff
@@ -512,6 +513,9 @@ impl ServoGlue {
                         PromptDefinition::Input(message, default, sender) => {
                             sender.send(cb.prompt_input(message, default, trusted))
                         },
+                        PromptDefinition::Credentials(_) => {
+                            warn!("implement credentials prompt for open harmony OS");
```

Comment:
@jdm should I use the `todo!()` macro instead of a warning?

Reply:
`todo!()` would cause servo to crash on ohos/android when a credential prompt is requested, that is definitely undesirable. A warning is perfectly fine I think.

Reply:
yeah that makes sense, regarding `todo!()`. Would it be worth marking `PromptDefinition` non-exhaustive now?

Reply:
> Would it be worth marking PromptDefinition non-exhaustive now?

I think it would be better to do that in a separate PR as it's unrelated to this one.

Reply:
Note: This code is shared and used by both OpenHarmony and Android, so probably the warning message should reflect that. 
Both currently don't support prompts, so just issuing a warning is perfectly fine for this PR.